### ISSUE TITLE
Add test RNA-Seq dataset?

### ISSUE THREAD

**Issue by LMBradford** [R-ODAF/R-ODAF_Health_Canada#167]
The current tests run on pushing to remote repos only have TempO-Seq data. Maybe we should have a second set of tests that use RNA-Seq data, to check that changes to the code work with both types.

**Comment by mattjmeier** [comment]
Agree, in fact we have discussed the idea in #3, but no progress on it really, yet! An important to-do.

**Comment by LMBradford** [comment]
I have a test dataset made, but we need to figure out how/when to implement it. Nothing is currently added to the test_data repo, but I will do so. Running RNA-seq data through the pipeline is much slower than TempO-seq because it takes so long to make the STAR and RSEM index files. So we probably don't want to have it run on every push. Is there a way to flag which workflows to run when giving the push command?

**Comment by LMBradford** [comment]
There's a test dataset on https://github.com/EHSRB-BSRSE-Bioinformatics/test-data/tree/main/rnaseq_singleend.
I've made a workflow to run the test, but I think it required too many resources to run. It also needs to be updated to reflect the improved env installation.
